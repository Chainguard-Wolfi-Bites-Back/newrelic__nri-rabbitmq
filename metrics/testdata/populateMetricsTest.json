{
    "name": "test-name",
    "vhost": "test-vhost",
    "messages_ready_details": {
        "rate": false
    }
}