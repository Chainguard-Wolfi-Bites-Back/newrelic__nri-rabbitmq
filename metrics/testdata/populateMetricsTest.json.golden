{"event_type":"testMetrics","queue.messagesReadyDeliveryClientsPerSecond":0}